### PR TITLE
DOCS-100 - Forward raw data

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request updates the following article: 
https://help.sumologic.com/docs/manage/data-forwarding/amazon-s3-bucket/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-100](https://sumologic.atlassian.net/browse/DOCS-100)

### CODE REVIEW

File: blog-service/2024-08-30-manage.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - manage
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We are happy to announce that you can now forward raw log data from Sumo Logic to an S3 destination. Previously, you could only forward raw log data together with its metadata and enriched fields. You can now choose to forward only log data, log data and metadata, or log data with metadata and enriched fields, in either CSV or JSON format.
+
+The ability to choose the kind of data forwarded, as well as its format in either CSV or JSON, allows you to perform more precise analytics on the data in your tools of choice.
```
Comment (kimsauce) [inline blog-service/2024-08-30-manage.md:15]:
```suggestion
We are happy to announce that you can now forward raw log data from Sumo Logic to an S3 destination. Previously, forwarding was limited to raw log data along with its metadata and enriched fields. Now, you have the flexibility to choose between forwarding only log data, log data with metadata, or log data with metadata and enriched fields, in either CSV or JSON format.

This enhanced flexibility enables you to perform more precise analytics on the data using your preferred tools.
```

File: docs/manage/data-forwarding/amazon-s3-bucket.md [hunk 2]
```diff
@@ -85,67 +85,74 @@ The file naming convention for legacy data forwarding (prior to January 2017) is
...
 1. **S3 Region**. Select the S3 region or keep the default value of Others. The S3 region must match the appropriate S3 bucket created in your Amazon account.
-1. **Enable S3 server-side encryption**. Select the check box if you want the forwarded data to be encrypted. For more information, see [Protecting Data Using Server-Side Encryption with Amazon S3-Managed Encryption Keys (SSE-S3)](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) in AWS help.
-1. **Active**. Select this check box to enable Data Forwarding for the entire S3 bucket. To start Data Forwarding you will also need to enable forwarding for the desired indexes, as described below in this topic.
+1. **Enable S3 server-side encryption**. Select the check box if you want the forwarded data to be encrypted. For more information, see [Using server-side encryption with Amazon S3 managed keys (SSE-S3)](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) in AWS help.
+1. **Active**. Select this check box to enable data forwarding for the entire S3 bucket. To start data forwarding you will also need to enable forwarding for the desired indexes, as described below.
```
Comment (kimsauce) [inline docs/manage/data-forwarding/amazon-s3-bucket.md:109]:
```suggestion
1. **Active**. Select this check box to enable data forwarding for the entire S3 bucket. To start forwarding data, you will also need to enable forwarding for the desired indexes, as described below.
```

File: docs/manage/data-forwarding/amazon-s3-bucket.md [hunk 3]
```diff
@@ -85,67 +85,74 @@ The file naming convention for legacy data forwarding (prior to January 2017) is
...
-   * `{index}` will be replaced by the name of the partition or scheduled view.
-   * `{day}` will be replaced by the day of the year in the yyyy-MM-dd format.
-   * `{hour}` will be replaced by the hour of the day (0-23).
-   * `{minute}` will be replaced by the minute of the hour.
-   * `{second}` will be replaced by the second of the minute.
-   * `{uuid}` will be replaced by a randomly generated universal unique identifier.<br/><br/>
-      If you leave this field blank, the default format `{index}_{day}_{hour}_{minute}_{second}` is used. For example, to place data in a directory named `SumoDataForwarding` you could specify the **File Format** as: `SumoDataForwarding/{day}/{index}_{day}_{hour}_{minute}_{second}`
+   * If you want to create a new destination, choose **New Amazon S3 Destination**, and follow the instructions in [Configure an S3 data forwarding destination](#configure-an-s3-data-forwarding-destination) above.
+   * If you want to use an existing destination, leave **Forwarding Destination** set to **Existing Amazon S3 Destination**, and click the pull-down to the right of **Amazon S3 Destination**, and pick a destination.
```
Comment (kimsauce) [inline docs/manage/data-forwarding/amazon-s3-bucket.md:129]:
```suggestion
   * If you want to use an existing destination, leave **Forwarding Destination** set to **Existing Amazon S3 Destination**. Then, click the drop-down menu next to **Amazon S3 Destination** select your desired destination.
```

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
Good suggestions, @kimsauce! Thanks!

**Pull request merged**